packages/block-editor/src/components/inner-blocks/use-nested-settings-update.js

```diff
@@ -98,7 +101,29 @@ export default function useNestedSettingsUpdate(
 		}
 
 		if ( ! isShallowEqual( blockListSettings, newSettings ) ) {
```

Comment:
What setting causes `isShallowEqual` to be false?

Reply:
none, it's just the first time this gets called, it needs to be batched across the 500 blocks that call it in the same render commit.

Reply:
Oh, ok. Why is that not initialised/preloaded in the store though?

Reply:
Sorry, am bit slow today. Because the information is in a React component of course.